config/sync/os2loop.settings.yml

```diff
@@ -16,3 +16,5 @@ search_settings:
     os2loop_subject: os2loop_subject
     os2loop_profession: 0
     os2loop_tag: 0
+os2loop_question:
+  enable_rich_text: 0
```

Comment:
This change is not related to media and should not be a part of this pull request.